noetic/distribution.yaml

```diff
@@ -9118,6 +9118,28 @@ repositories:
       url: https://github.com/ros-geographic-info/unique_identifier-release.git
       version: 1.0.6-1
     status: maintained
+  uos_tools:
+    doc:
+      type: git
+      url: https://github.com/uos/uos_tools.git
+      version: master
+    release:
+      packages:
+      - uos_common_urdf
+      - uos_diffdrive_teleop
```

Comment:
This package appears to be the one with GPL content in it not the uos_tools metapackage below. Which does not match the package.xml declaration of BSD

https://github.com/uos/uos_tools/search?q=gnu


https://github.com/uos/uos_tools/blob/0b12326ec37edffac0799724cf2c8d54facd8129/uos_diffdrive_teleop/package.xml#L7